task(teaching-stage): Add Self-Study part (#22)
open-education-hub/methodology: description.md

### Self-Study

Self-Study is the process of independently learning outside of structural classroom settings or formal instruction.

There are multiple reasons students find self-studying hard:

- **Lack of structure**: Learning without a structure can lead to potential confusion and inefficiency.

> - **Access to valuable resources**: Even if there are a lot of resources on the internet, they are not always well structured.

Reviewer: ```suggestion
- **Access to resources**: Even if there are a lot of resources on the internet, they are not always easy to find.
```
**valuable** is subjective. I suggested something for the argumentation so "structure" will not be repeated.

- **Lack of feedback**: Without regular feedback from the teacher, students tend to lose motivation, potentially leading to gaps in knowledge.
- **Need for help**: Without help from the teacher, students may not be able to solve the problems on their own and this can lead to frustration.
- **Difficulty**: The subjects must have an adequate difficulty in order not to demotivate students.

In order to overcome these problems, we can improve the following:

#### Organize resources

> The material is to be organized into distinct chapters.

Reviewer: ```suggestion
The material is to be organized into chapters.
```

Each chapter will encompass one or more subjects.
Within each chapter, there will be an `overview` section that summarizes the different subjects included.

> This `overview` will highlight the main topics of the chapter and the key information that readers should grasp before moving on to the next chapter.

Reviewer: You could also add a "requirements" section at the beginning of each chapter.

A general example of the following structure is shown below or can be found [here](https://github.com/open-education-hub/oer-template).

> ```bash

Reviewer: ```suggestion
```console
```
I do not remember why, but at some point, we decided that `console` >>> `bash`. @teodutu do you remember why?

└── chapters/
    ├── chapter-1/

>     │   ├── overview

Reviewer: ```suggestion
    │   ├── overview.md
```

Reviewer 2: @gabrielmocanu , I'm leaving this, but note that it shouldn't be here. It's content-centric, when it should be content delivery-centric.

    │   ├── subject-1
    │   └── subject-2
    └── chapter-2/
        ├── overview
        ├── subject-1
        └── subject-2
```

#### Drills and quizzes

> To improve the learning experience and for the students to have a method to test their knowledge, drills and quizzes can be used.

Reviewer: ```suggestion
To improve the learning experience and for the students to have a method to test their knowledge, [drills](#drills) and [quizzes](#quizzes) can be used.
```
As one old man said (probably Yoda): **link, always link**.
